ReactWindows/ReactNative/Modules/AccessibilityInfo/AccessibilityInfoModule.cs

```diff
@@ -0,0 +1,77 @@
+﻿using ReactNative.Bridge;
+using ReactNative.Modules.Core;
+using System.Collections.Generic;
+using Windows.UI.ViewManagement;
+
+namespace ReactNative.Modules.Accessibilityinfo
+{
+    class AccessibilityInfoModule : ReactContextNativeModuleBase
+    {
+        private readonly AccessibilitySettings _accessibility = new AccessibilitySettings();
+        private readonly UISettings _settings = new UISettings();
+
+        private string GetRgbaString(UIElementType type)
+        {
+            var color = _settings.UIElementColor(type);
+            return "rgba(" + color.R + "," + color.G + "," + color.B + "," + color.A + ")";
+        }
+
+        private IDictionary<string, string> GetHighContrastColors()
```

Comment:
>private IDictionary<string, string> GetHighContrastColors() [](start = 8, length = 59)

Maybe put a TODO about how we want to expose these colors to JavaScript in the future

Reply:
Added